main.cpp

```diff
@@ -66,9 +66,27 @@ void variableDeclarations()
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
     
-    
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    int shoeSize = 10; 
+    int waveform = 1;
+    int count = 0;
+
+    float dial = 0.0f;
+    float footLength = 30.5f;
+    float height = 1.959484f;
+
+    double sample = 0.565456456012345;
+    double frequency = 103.123;
+    double amplitude = 0.5954;
+
+    char preset = 'A';
+    char key = 71;
+    char aisle = 'f';
```

Comment:
I can see that you understand how to initialize `char` types.
just be aware that they should not be initialized with numbers directly like this. 
a `char` is meant to represent an alpha-numeric.  So, we should convey that intent by initializing them with alpha-numerics only.